### PR TITLE
Added lookup V2 endpoint + example

### PR DESCRIPTION
**Pull request by RyapovDimitar**
Added lookup V2 endpoint + example

### CODE REVIEW

File: Experian Address Validation.postman_collection.json [hunk 1]
```diff
@@ -1031,6 +1032,128 @@
...
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "Accept",
+								"value": "application/json"
+							},
+							{
+								"key": "Add-Addresses",
+								"value": "true",
```
Comment (keelimbeh) [inline Experian Address Validation.postman_collection.json:1070]: Suggest to add description

File: Experian Address Validation.postman_collection.json [hunk 2]
```diff
@@ -1031,6 +1032,128 @@
...
+									"mode": "raw",
+									"raw": "{\n  \"country_iso\": \"GBR\",\n  \"max_addresses\":250,\n  \"datasets\": [\n    \"gb-address\"\n  ],\n  \"key\": {\n    \"type\": \"locality\",\n    \"value\": \"London\"\n  }\n}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{baseUrl}}/address/lookup/v1",
```
Comment (keelimbeh) [inline Experian Address Validation.postman_collection.json:1132]: should be v2

File: Experian Address Validation.postman_collection.json [hunk 3]
```diff
@@ -1031,6 +1032,128 @@
...
+									"host": [
+										"{{baseUrl}}"
+									],
+									"path": [
+										"address",
+										"lookup",
+										"v1"
```
Comment (keelimbeh) [inline Experian Address Validation.postman_collection.json:1139]: should be v2

**Review by jbate — CHANGES_REQUESTED** [review]
As already mentioned, there appear to be some instances of "v1" appearing in the json.

**Review by wnkr — APPROVED** [review]

File: Experian Address Validation.postman_collection.json (hunk 1 above)
Comment (RyapovDimitar) [reply Experian Address Validation.postman_collection.json:1071]: Added description

File: Experian Address Validation.postman_collection.json (hunk 3 above)
Comment (RyapovDimitar) [reply Experian Address Validation.postman_collection.json]: Fixed

**Pull request merged**